Python/Quantum_random/Quant_random.py

```diff
@@ -0,0 +1,41 @@
+import qiskit as qk
+import warnings
+import os
+
+#Load your account if running locally
+# from qiskit import IBMQ
+# IBMQ.save_account('YOUR_API_KEY')
```

Comment:
Should I include an image? I think that there was an issue to remove the images from the repo.

Reply:
Can you also explain to me how should I include the .env file update like should I push the .env.example with the commented commands?

Reply:
Sure @RutvikJ77 

Let me explain this part:

- Have a `.env` file in your Directory and add the following: 

```
API_KEY = "XXXXXXXXXXXXXXXXXXXXXXX"
```

- Install a Package named `decouple` to get the Environment Variables. Import it like this: 

`
from decouple import config
`

- Go to your Main Script and add the following line of code: 

`
api_key = config("API_KEY")
`

You are now ready to use the API Key in your Code. Do not push the `.env` in your Code because its not a good practise. Have a `.env.example` which the User can rename to `.env` and store their API Key. Have the `.env.example` for sure.

Reply:
> Should I include an image? I think that there was an issue to remove the images from the repo.

You can add Images. Just upload the Image on the Imgur and reference it on the README

Reply:
Cool! That should do, Thanks for the clarity. If I do not go through decouple is it okay? I have another way as well. It basically takes an argument of the API key. We can use `from os import environ` which should do the work. Though I will also do the decouple to understand it better.

Reply:
Sure. Feel free to use whatever you find fit